bin/handlerFunctions.js

```diff
@@ -214,6 +214,81 @@ async function editReg(argv) {
     }
 }
 
+// Set the Power Save value
+async function setPowerSave(argv) {
+    // get player data from argv
+    let playerData;
+    try {
+        playerData = await pullData(argv);
+        // playerData[0] = playerUser, [1] = playerIP, [2] = playerPW
+    } catch (err) {
+        errorHandler(err);
+        return;
+    }
+
+    logIfOption('Editing Power Save mode for ' + argv.playerName  + ' ' + argv.connector + ' ' + argv.device + ' to ' + argv.enabled, argv.verbose);
+
+    // create body
+    let rawBody = JSON.stringify({ "enabled": argv.enabled });
+
+    let requestOptions = {
+        method: 'PUT',
+        url: 'http://' + playerData[1] + '/api/v1/video/' + argv.connector + '/output/' + argv.device + '/power-save',
+        headers: { 'Content-Type': 'application/json' },
+        body: rawBody
+    }
+
+    logIfOption('Sending ' + requestOptions.method + ' request to ' + requestOptions.url, argv.verbose);
+
+    // send request
+    try {
+        let response = await requestFetch(requestOptions, playerData[0], playerData[2]);
+        logIfOption('Response received! => ', argv.verbose);
+        if (!argv.rawdata) {
+            console.log('Power Save mode changed: ' + response.data.result);
+        } else if (argv.rawdata) {
+            console.log(response.data.result);
+        }
+    } catch (err) {
+        errorHandler(err);
+    }
+}
+
+// Get the Power Save value
+async function getPowerSave(argv) {
+    // get player data from argv
+    let playerData;
+    try {
+        playerData = await pullData(argv);
+        // playerData[0] = playerUser, [1] = playerIP, [2] = playerPW
```

Comment:
remove

Reply:
Updated in https://github.com/brightsign/player-cli/pull/23/commits/91ba4af98ae55584cce43c3e741d7a8fb49e674c.